bundle/config/validate/all_resources_defined.go

```diff
@@ -0,0 +1,47 @@
+package validate
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+)
+
+func AllResourcesHaveValues() bundle.Mutator {
+	return &allResourcesHaveValues{}
+}
+
+type allResourcesHaveValues struct{}
+
+func (m *allResourcesHaveValues) Name() string {
+	return "validate:all_resources_have_values"
+}
+
+func (m *allResourcesHaveValues) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+	rv := b.Config.Value().Get("resources")
```

Comment:
you can call b.Config.Mutate instead, to be consistent with the rest of mutators even though you're not mutating dynamic value

Reply:
done

Reply:
On further thought, I reverted this back to `b.Config.Value()`. I believe the current approach might be better because we don't need to modify the configuration tree; we just read values from it and emit an error.